syntax = "proto3";

package astre.ecs;

message CameraComponent 
{
    float fov = 1;          // Field of view (degrees)
    float near_plane = 2;   // Near clipping plane
    float far_plane = 3;    // Far clipping plane
    float aspect = 4;       // camera aspect ratio
}